oaknational/Oak-Web-Application
# feat(security): add security disclosure policy

## Description

This adds support for the security.txt standard. 

It includes:

1. security.txt policy
2. security disclosure policy
3. PGP key for encrypted email communications

## How to test

1. Go to /.well-known/security.txt
2. You should see our security.txt policy

1. Go to /.well-known/pgp-key.txt
2. You should see our pgp key

1. Go to legal/security-disclosure-policy
2. You should see our security disclosure policy

## Screenshots

How it should now look:
<to be added>

## Checklist
